src/averageAges.js

```diff
@@ -37,7 +40,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womans = people.filter((woman) =>
+    woman.sex === 'f'
+    && (withChildren === true
```

Comment:
If i understand correctly, the withChildren variable is of boolean type. You can omit the comparison. The expression by itself will be true when it's true and false when it's false.
```suggestion
    && (withChildren
```